src/test/java/teammates/test/cases/ui/browsertests/AdminHomePageUiTest.java

```diff
@@ -44,6 +45,7 @@
 import teammates.test.pageobjects.StudentHomePage;
 import teammates.test.pageobjects.StudentProfilePage;
 import teammates.test.util.Priority;
+import teammates.ui.controller.AdminInstructorAccountAddAction;
```

Comment:
Maybe better if you just static import the constant, or just move the constant to Const, but you decide.